ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -433,16 +424,7 @@ protected Node visitCreateTable(final CreateTable node, final Object context) {
         .map(tableElement -> (TableElement) process(tableElement, context))
         .collect(Collectors.toList());
 
-    final Map<String, Literal> rewrittenProps = node
-        .getProperties()
-        .entrySet()
-        .stream()
-        .collect(Collectors.toMap(
-            Entry::getKey,
-            e -> (Literal) process(e.getValue(), context)
-        ));
-
-    return node.copyWith(rewrittenElements, rewrittenProps);
+    return node.copyWith(rewrittenElements, node.getProperties());
```

Comment:
This is no longer allowing the literals to be re-written. Is this intentional?

Reply:
See above